src/cmd_all/src/bin/risingwave.rs

```diff
@@ -128,6 +130,7 @@ impl Component {
             Component::Compactor => vec!["compactor-node", "compactor_node"],
             Component::Ctl => vec!["risectl"],
             Component::Playground => vec!["play"],
+            Component::AllInOne => vec!["all-in-one"],
```

Comment:
nit: Should we come up with another name like “monolithic mode” as mentioned in https://github.com/risingwavelabs/risingwave/issues/10908#issuecomment-1632211486? Don’t know if there are any other conventions for reference.

Reply:
As “all-in-one” is also for all-in-one binary, and sounds a little … maybe informal? 🤣